azure-devops/azext_devops/dev/common/artifacttool_updater.py

```diff
@@ -124,9 +125,22 @@ def _update_artifacttool(uri, release_id):
 
             # Move the release into the real releases location
             release_dir = _compute_release_dir(release_id)
-            logger.debug("Moving downloaded ArtifactTool from %s to %s", release_temp_dir, release_dir)
-            os.rename(release_temp_dir, release_dir)
-            logger.info("Downloaded Universal Packages tooling successfully")
+            if os.path.exists(release_dir):
```

Comment:
Can it end up in a situation where directory exists but it is empty. The command will not even try to put the tool in the directory in that case.

Reply:
That shouldn't happen unless the download of the tool itself fails, in which case this will fail much earlier.